docpages/example_programs/interactions_and_components/modal_dialog_interactions.md

```diff
@@ -29,9 +29,21 @@ int main(int argc, char const *argv[])
 			/* Add a text component */
 			modal.add_component(
 				dpp::component().
-				set_label("Type rammel").
+				set_label("Short type rammel").
 				set_id("field_id").
 				set_type(dpp::cot_text).
+				set_placeholder("gumd").
+				set_min_length(5).
+				set_max_length(50).
+				set_text_style(dpp::text_short)
+			);
+			/* Add another text component in the next row, as required by Discord */
+			embed.add_row();
```

Comment:
should be modal.add_row() not embed.add_row() surely?

Reply:
solved.